spec/models/form_submission_attempt_spec.rb

```diff
@@ -60,6 +60,52 @@
 
           expect(SimpleFormsApi::NotificationEmail).not_to have_received(:new)
         end
+
+        context 'is a form526_form4142 form' do
+          let(:vanotify_client) { instance_double(VaNotify::Service) }
+          let!(:email_klass) { EVSS::DisabilityCompensationForm::Form4142DocumentUploadFailureEmail }
+          let!(:form526_submission) { create(:form526_submission) }
+          let!(:form526_form4142_form_submission) do
+            create(:form_submission, saved_claim_id: form526_submission.saved_claim.id,
```

Comment:
Don't need to chain `.id`. Can simply call `saved_claim_id` on `form_submission`.